tests/Doctrine/Tests/TestInit.php

```diff
@@ -1,17 +0,0 @@
-<?php
-/*
- * This file bootstraps the test environment.
- */
-
-error_reporting(E_ALL | E_STRICT);
-date_default_timezone_set('UTC');
```

Comment:
that doesn't belong to the test setup. it's random a random value that has nothing to to with the tests. for testing locally its part of the php setup otherwise it would fail for every project of those people anyway. not just dbal.